components/gitpod-db/src/periodic-deleter.ts

```diff
@@ -36,6 +36,9 @@ export class PeriodicDbDeleter {
             const toBeDeleted: { table: string; deletions: string[] }[] = [];
             for (const table of sortedTables) {
                 const rowsForTableToDelete = await this.collectRowsToBeDeleted(table);
+                if (rowsForTableToDelete.deletions.length === 0) {
```

Comment:
Possibly better flow is this:
Do not log the # of rows, only log at the end but we attach that info onto the log entry.

This means:
* We get a confirmation log that it completed (which we miss in this case, so we don't know if it's still running, or if it in fact completed)
* We still log roughly the same amount

Reply:
> We get a confirmation log that it completed

That's logged down bellow:

> [PeriodicDbDeleter] Finished deleting records.

Reply:
I know, the point was that we'd not log on each table, but once in aggregate for all tables, including when it deleted 0 entries for a table.